src/views/product-downloads-view/components/featured-tutorials-section/index.tsx

```diff
@@ -24,24 +22,7 @@ const FeaturedTutorialsSection = ({
 			>
 				Featured Tutorials
 			</Heading>
-			<CardsGridList>
-				{featuredLearnCards.map((cardProps: FeaturedLearnCard) => {
-					const { id, type } = cardProps
-					if (type == 'collection') {
```

Comment:
I think we want to retain this one because of the collection cards?

Reply:
This section no longer renders collection cards as of #841! 😊

Reply:
Oh okay! It looks like `FeaturedLearnCard` still shows that the[ collection type ](https://github.com/hashicorp/dev-portal/blob/main/src/views/product-downloads-view/types.ts#L40-L42)is acceptable. Maybe we should make a follow up task to update that type?